lib/large_object_store.rb

```diff
@@ -10,7 +10,8 @@ def self.wrap(store)
   class RailsWrapper
     attr_reader :store
 
-    LIMIT = 1024**2 - 100
+    MAX_OBJECT_SIZE = 1024**2
+    ITEM_HEADER_SIZE = 100
```

Comment:
I saw a `sizeof(item)` in the memcached source code so depending on C compiler struct alignment, the header size could possibly vary ... unlikely, but, yeah didn't seem like there was anything to be gained in reducing this number!